curriculum/challenges/english/25-front-end-development/quiz-css-positioning/66ed8ffcf45ce3ece4053eb5.md

```diff
@@ -628,3 +628,527 @@ Which of the following is the default value of the `position` property?
+  top: 0;
+  left: 0;
+}
+```
+
+---
+
+```css
+.box {
+  position: static;
+}
+```
+
+#### --answer--
+
+```css
+.box {
+  position: relative;
+  top: 10px;
+  left: 10px;
+}
+```
+
+### --question--
+
+#### --text--
+
+What happens to an element with `position: static`?
+
+#### --distractors--
+
+It is removed from the normal document flow.
+
+---
+
+It overlaps other elements.
+
+---
+
+It sticks to the viewport during scrolling.
+
+#### --answer--
+
+It follows the normal document flow.
+
+### --question--
+
+#### --text--
+
+Which of the following is true about `position: absolute`?
+
+#### --distractors--
+
+It positions the element relative to the viewport.
+
+---
+
+It keeps the element in the normal document flow.
+
+---
+
+It positions the element relative to the document root.
+
+#### --answer--
+
+It positions the element relative to its nearest positioned ancestor.
+
+### --question--
+
+#### --text--
+
+What is the difference between `position: fixed` and `position: sticky`?
+
+#### --distractors--
+
+`fixed` positions the element relative to its parent, while `sticky` positions it relative to the viewport.
+
+---
+
+`fixed` keeps the element in the normal flow, while `sticky` removes it.
+
+---
+
+`fixed` allows scrolling, while `sticky` does not.
+
+#### --answer--
+
+`fixed` positions the element relative to the viewport, while `sticky` positions it relative to its nearest scrollable ancestor.
+
+### --question--
+
+#### --text--
+
+Which of the following is an example of using `z-index`?
+
+#### --distractors--
+
+```css
+.box {
+  position: static;
+  z-index: 10;
+}
+```
+
+---
+
+```css
+.box {
+  position: relative;
+}
+```
+
+---
+
+```css
+.box {
+  position: absolute;
+}
+```
+
+#### --answer--
+
+```css
+.box {
+  position: absolute;
+  z-index: 5;
+}
+```
+
+### --question--
+
+#### --text--
+
+What is the default stacking context in CSS?
+
+#### --distractors--
+
+The `body` element.
+
+---
+
+The `html` element.
+
+---
+
+The `viewport`.
+
+#### --answer--
+
+The root element of the document.
+
+### --question--
+
+#### --text--
+
+Which of the following is true about `position: sticky`?
+
+#### --distractors--
+
+It positions the element relative to the viewport.
+
+---
+
+It removes the element from the normal flow.
+
+---
+
+It keeps the element fixed at all times.
+
+#### --answer--
+
+It positions the element relative to its nearest scrollable ancestor and becomes fixed when scrolling past a threshold.
+
+### --question--
+
+#### --text--
+
+What is the effect of `position: relative` on an element?
+
+#### --distractors--
+
+It removes the element from the normal flow.
+
+---
+
+It positions the element relative to the viewport.
+
+---
+
+It keeps the element fixed during scrolling.
+
+#### --answer--
+
+It offsets the element from its normal position without removing it from the document flow.
+
+### --question--
+
+#### --text--
+
+Which of the following is an example of using `position: fixed`?
```

Comment:
this question should be updated because all of the distractors are not using fixed which makes it to easy for the obvious right answer